### ISSUE TITLE
Wishlist for demos

### ISSUE THREAD

**Issue by olexandr-konovalov** [OpenDreamKit/gap-demos#1]
Which other demos would be good to have here?
- [x] Francy
- [x] JupyterViz
- [x] Meataxe64
- [x] HPC-GAP

**Comment by stevelinton** [comment]
Add `syntaxtree`. It should be possible to demo some kind of very crude automatic parallelisation, turning 

```
for i in [1..n] do
    y[i] := foo(x[i]);
od;
```

into 
```
tasks := [];
for i in [1..n] do
   tasks[i] := RunTask(foo,x[i]);
od;
for i in [1..n] do
   y[i] := TaskResult(tasks[i]);
od;
Unbind(tasks);
```
or something like that. 

Maybe also include something with an SCSCP server farm.

**Comment by olexandr-konovalov** [comment]
Add SemigroupViz demo - this is https://github.com/nathancarter/semigroupviz by @nathancarter, not [SgpViz](https://www.gap-system.org/Packages/sgpviz.html).

**Comment by nthiery** [comment]
At this rate, it will soon be nice to run [planetaryum](https://github.com/OpenDreamKit/planetaryum) on this repo :-)

**Comment by olexandr-konovalov** [comment]
Suggested by @stevelinton in D5.15: MajoranaAlgebras by Markus Pfeiffer and Madeleine Whybrow, which constructs Majorana representations of finite groups (added in GAP4.10.1).

**Comment by olexandr-konovalov** [comment]
Pushed very much initial version of JupyterViz demo with an incomplete map of locations of GAP package authors and some heuristics implemented to determine countries if they are not stated in PackageInfo.g

![Screenshot 2019-10-21 at 00 55 57](https://user-images.githubusercontent.com/5751387/67168577-d3c79c00-f39d-11e9-81b9-a88f26f88fcf.png)

**Comment by nathancarter** [comment]
Nice!!

**Comment by olexandr-konovalov** [comment]
Thanks @nathancarter! I will update the data cleaning script and offload most of it to a `.g` file, but if you have any suggestions how the map could be made better, they will be much appreciated!

**Comment by olexandr-konovalov** [comment]
@stevelinton for HPC-GAP in Jupyter see https://github.com/gap-packages/JupyterKernel/issues/108. Getting it to work requires https://github.com/gap-system/gap/pull/3710 and https://github.com/gap-packages/ZeroMQInterface/pull/26. `Print` does not work though.

**Comment by olexandr-konovalov** [comment]
@nathancarter is there an option to put different marks on the line plot (instead of circles, use also triangles and squares)? We want it for the plot at the bottom of https://github.com/OpenDreamKit/gap-demos/blob/master/notebooks-master/Meataxe64%20Demo.ipynb

**Comment by nathancarter** [comment]
Yes.  To specify detailed options, you have to dig inside a bit, but it goes like so:

```gap
LoadPackage( "jupyterviz" );

# let's assume you have your list of numbers to plot here...
numbers := [ 3, 5, 4, 6, 5, 7 ];

# we package it up for use by the Plotly graphing tool:
series := ConvertDataSeriesForTool.plotly( [
    JUPVIZMakePlotDataSeries( numbers ) ] );

# then we tweak one of Plotly's defaults, to make the markers different:
series.data[1].marker := rec( size := 20, symbol := "cross-dot" );
# (see https://plot.ly/javascript/reference/#scatter-marker-symbol
# for all the different types of markers you can use)

# then we have JupyterViz draw it, using Plotly:
CreateVisualization( rec( tool := "plotly", data := series ) );
```

Does that do what you want?
